tests/meta/orders/test_meta_orders_api.py

```diff
@@ -405,20 +404,20 @@ def test_validate_sex_inconsistent_sex(
     # WHEN calling _validate_sex
     # THEN an OrderError should be raised on non-matching sex
     with pytest.raises(OrderError):
-        validator._validate_subject_sex(samples=order_data.samples, customer_id=order_data.customer)
+        validator._validate_subject_sex(samples=samples, customer_id=order_data.customer)
 
 
 def test_validate_sex_consistent_sex(
     orders_api: OrdersAPI, mip_order_to_submit: dict, helpers: StoreHelpers
 ):
     # GIVEN we have an order with a sample that is already in the database and with same gender
-    order_data = OrderIn.parse_obj(mip_order_to_submit, project=OrderType.MIP_DNA)
+    order_data = MipDnaOrder.model_validate(mip_order_to_submit)
     store = orders_api.status
     customer: Customer = store.get_customer_by_internal_id(customer_internal_id=order_data.customer)
 
     # add sample with different sex than in order
-    sample: MipDnaSample
-    for sample in order_data.samples:
+    samples: list[Sample] = [sample for _, _, sample in order_data.enumerated_new_samples]
```

Comment:
and probably same suggestion here

Reply:
A bit uncertain of what suggestion you meant?

Reply:
nevermind, already done